### DISCUSSION TITLE
Several automations at the same time

### DISCUSSION THREAD

**Discussion by rafikapp212** [Budibase/budibase#13402] [Help] [answered]
Hello everyone.
It's been a while since I worked with budibase, and I had a problem which was that I couldn't run two automations at the same time. for example I have a automation which is going to execute some instructions and it has a delay of 10 min to finish its execution, so during this time (10 min) I cannot execute another automaton until the first one has yet finished.

So I just want to know if this problem has been resolved or not.
Thank you.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @rafikapp212,

I believe I answered this for you on our discord. Just going to post the link to the discussion as I think i've answered this for you. https://discord.com/channels/733030666647765003/1225383044521660466/1225383044521660466
